server/pbench/bin/pbench-sync-satellite

```diff
@@ -151,14 +154,19 @@ for host in $hosts ;do
         mkdir -p $logdir/$host
     fi
 
+    # get the tarball list for this host
+    flist=$(find $unpack/$host -type f -name '*.tar.xz.md5' | sed 's;'$unpack/$host/';;')
+
+    echo $flist
+    
     # move the unpacked files from tmp directory to archive
     mv $unpack/$host/* $unpack/$host/.prefix* .
```

Comment:
when we use prefix this will not work. Rather we can do something like:
mv $unpack/$host/* .
mv $unpack/$host/.prefix/* .prefix

Reply:
Right - fix coming up shortly.